deploy/scripts/000039-upgrade-network-info-v2.ts

```diff
@@ -0,0 +1,39 @@
+import { DeployedContracts, InstanceName, setDeploymentMetadata, upgradeProxy } from '../../utils/Deploy';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
+    const { deployer } = await getNamedAccounts();
+
+    const network = await DeployedContracts.BancorNetworkV1.deployed();
```

Comment:
thees should use the latest versions

Reply:
Yeah. Deleting older deployments would simplify this...

Reply:
👍